config.js

```diff
@@ -992,6 +992,10 @@ var config = {
         // all SDPs with an empty string instead.
         // rtcstatsSendSdp: false,
 
+        // This determines if rtcstats from lib-jitsi-meet will be used instead of the one
+        // here in jitsi-meet directly
+        // rtcstatsUseLibJitsi: false,
```

Comment:
Let's move this to `testing` since it's temporary...

Reply:
Done

Reply:
Done.